app/src/main/java/chat/rocket/android/chatroom/viewmodel/ViewModelMapper.kt

```diff
@@ -303,4 +307,16 @@ class ViewModelMapper @Inject constructor(private val context: Context,
             })
         }
     }
-}
\ No newline at end of file
+}
+
+internal val FileAttachment.attachmentTitle: String
```

Comment:
I'm not sure if it makes sense using the `internal` modifier on any class from the app project.

Reply:
True...